lib/progeny/command.rb

```diff
@@ -228,7 +229,7 @@ def exec!
     #   exceeds the amount specified by the max argument.
     def read_and_write(input, stdin, stdout, stderr, timeout=nil, max=nil)
       max = nil if max && max <= 0
-      @out, @err = '', ''
+      @out, @err = +'', +''
```

Comment:
> +@: Returns a string that is not frozen: self, if not frozen; self.dup otherwise.

https://ruby-doc.org/3.2.2/String.html#class-String-label-Methods+for+a+Frozen-2FUnfrozen+String